src/snyk/types.ts

```diff
@@ -1,4 +1,39 @@
-// See https://snyk.docs.apiary.io/#reference/projects/all-projects/list-all-projects
+// See https://apidocs.snyk.io/?version=2023-08-04#get-/orgs/-org_id-/projects
+export interface ProjectResponse {
+  data: RestAPIProject[]
+  links: {
+    next?: string
+  }
+}
+
+export interface RestAPIProject {
+  id: string
+  attributes: {
+    name: string
+    type: string
+    origin: string
+    created: string
+    status: string
+    settings: {
+      recurring_tests: {
+        frequency: string
+      }
+    }
+  }
+  status: boolean
+  meta: {
+    latest_dependency_total: {
+      updated_at: string
+      total: number
+    }
+    latest_issue_counts: {
+      critical?: number
+      high: number
+      medium: number
+      low: number
+    }
+  }
+}
```

Comment:
Could you add a comment (link and/or description) that makes it clear that the SnykProject type represents responses from the deprecated List all projects v1 API?